src/styles/styles.js

```diff
@@ -3533,6 +3590,60 @@ const styles = {
         textAlign: 'center',
     },
 
+    loginButtonRow: {
+        justifyContent: 'center',
+        width: '100%',
+        ...flex.flexRow,
+    },
+
+    loginButtonRowSmallScreen: {
+        justifyContent: 'center',
+        width: '100%',
+        marginBottom: 10,
+        ...flex.flexRow,
+    },
+
+    appleButtonContainer: {
+        width: 40,
```

Comment:
Instead of hardcoding 40 here, maybe we should use `variables.iconSizeExtraLarge`

Reply:
maybe also create another style:

```
iconExtraLarge: {
    width: variables.iconSizeExtraLarge,
    height: variables.iconSizeExtraLarge,
}
```

Then where this is used just do `[styles.iconSizeExtraLarge, styles.mr5]`. I think is more transparent to anyone reading the component as well and helps keep our styles more DRY. The `arrowIcon` also has some overlap here.

Reply:
This might be a bad merge, this code is not related to the feature I added.. I will look into it.

Reply:
Removed this code as it wasn't related to my code change.